utils/src/test/kotlin/UtilsTest.kt

```diff
@@ -29,11 +29,11 @@ import java.nio.file.Paths
 class UtilsTest : WordSpec({
     "disjoint" should {
         "return true for collections that have no common elememts" {
```

Comment:
Return type is not boolean anymore, please adjust the test name accordingly.

Reply:
Done.